docs/reference/glossary.asciidoc

```diff
@@ -24,28 +24,46 @@ Also see <<glossary-text,text>> and <<glossary-term,term>>.
 // end::analysis-def[]
 --
 
+[[glossary-auto-follow-pattern]] auto-follow pattern ::
+// tag::auto-follow-pattern-def[]
+An index pattern pattern used to automatically configure new indices as 
+<<glossary-follower-index, follower indices>> for <<glossary-ccr,{ccr}>>.
+// end::auto-follow-pattern-def[]
+
 [[glossary-cluster]] cluster ::
 // tag::cluster-def[]
-A cluster consists of one or more <<glossary-node,nodes>> which share the
-same cluster name. Each cluster has a single master node which is
-chosen automatically by the cluster and which can be replaced if the
-current master node fails.
+One or more <<glossary-node,nodes>> that share the
+same cluster name. Each cluster has a single master node, which is
+chosen automatically by the cluster and can be replaced if it fails.
 // end::cluster-def[]
 
+[[glossary-cold-phase]] cold phase ::
+// tag::cold-phase-def[]
+The third possible phase in the lifecycle of an index. 
```

Comment:
I'm not sure "third" is super helpful here without more context. Also another good opportunity for an external link.
> The third possible phase in the lifecycle of an index.

Reply:
I'm going to stet this to leave the emphasis on lifecycle, but will add the link. I think "third" is somewhat helpful, in that it indicates that it's some way along in the lifecycle.